### PR TITLE
Ft support before/after lstm

### PR DESCRIPTION
**Pull request by Or-Tal**
I've added support in demucs and in autoencoder model note that if you wish to take features before the attn module you should add get_ft_after_lstm: False under the features model configurations

### CODE REVIEW

File: denoiser/batch_solvers/batch_solver_factory.py [hunk 1]
```diff
@@ -44,7 +45,8 @@ def get_bs(args):
             return AdversarialBS(args, generator, discriminator, ft_config)
         else:
             if args.experiment.model == "demucs":
-                generator = Demucs(DemucsConfig(**args.experiment.demucs), include_ft_in_output=include_ft)
+                generator = Demucs(DemucsConfig(**args.experiment.demucs), include_ft_in_output=include_ft,
+                                   get_ft_after_lstm=get_ft_after_lstm)
```
Comment (mosem) [inline denoiser/batch_solvers/batch_solver_factory.py:49]: Shouldn't the include_ft_in_output and get_ft_after_lstm paramers be part of the parameters that the DemucsConfig holds?

**Review by mosem — APPROVED** [review]
I have one comment, other than that looks good.
Is there more code that demonstrates the different usages? before/after lstm/attn module?

File: denoiser/batch_solvers/batch_solver_factory.py (hunk 1 above)
Comment (Or-Tal) [reply denoiser/batch_solvers/batch_solver_factory.py:49]: Agree.. I pass a conditioning module in next pull requests that hold these functionalities

**Comment by Or-Tal** [comment]
I didn't understood your last comment

**Pull request merged**